rsrc/dialogs/preferences.xml

```diff
@@ -43,6 +43,7 @@
 	</group>
 	<text name='misc-head' size='large' relative='pos-in pos' anchor='spd-head' top='30' left='0' width='182' height='17'>Miscellaneous:</text>
 	<led name='nosound' relative='pos-in pos' anchor='misc-head' top='6' left='15'>No Sounds</led>
+	<led name='keyshift' relative='pos-in pos' rel-anchor='prev' top='10' left='0'>Directional keys can shift the screen</led>
```

Comment:
I think this should make it more clear that it's swapping one capability with another. Something like "Directional keys shift… instead of…".

Reply:
Better, but I think a different word than "squares" should be used. Maybe "tiles" or "spaces"?

Reply:
done and done

Reply:
@CelticMinstrel What if holding shift performed the opposite function so both are always available?

Reply:
The dialog could have a parenthetical text explaining this

Reply:
Sure, why not. The only question would be which modifier to pick… what precedent do we have so far? There's also the option of Control/Command (that is, Command on Mac, Control on non-Mac).

Given how "shift" typically means "extend", it seems like that doesn't really fit this kind of use. I guess "alt" fits better. Not sure whether "control" fits.

Reply:
replied to this out-of-thread